### DISCUSSION TITLE
planning dwdm network via oopt-gnpy?

### DISCUSSION THREAD

**Discussion by buts101** [Telecominfraproject/oopt-gnpy#487] [Q&A]
Is it possible to plan dwdm network with oopt-gnpy like optiwave or vendor software?

**Comment by EstherLerouzic** [comment]
Dear @buts101,

you can plan DWDM felxi grid networks using GNPy, using the gnpy-path-request with a list of service. The result include the path, mode selection, feasibility evalutation and spectral assignment for each service. I am not very familiar with optiwave software, so I don't know if this is similar.
